src/pydata_sphinx_theme/locale/fr/LC_MESSAGES/sphinx.po

```diff
@@ -2,33 +2,33 @@
 # Copyright (C) 2023 PyData developers
 # This file is distributed under the same license as the pydata-sphinx-theme
 # project.
-#
+# 
 # Translators:
 # Rambaud Pierrick <pierrick.rambaud49@gmail.com>, 2024
+# Denis Bitouzé <dbitouze@wanadoo.fr>, 2024
+# 
 msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2024-03-11 13:22-0500\n"
+"POT-Creation-Date: 2024-04-29 13:43+0200\n"
 "PO-Revision-Date: 2023-04-14 14:57+0000\n"
-"Last-Translator: Rambaud Pierrick <pierrick.rambaud49@gmail.com>, 2024\n"
-"Language: fr\n"
-"Language-Team: French "
-"(https://app.transifex.com/12rambau/teams/166811/fr/)\n"
-"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % "
-"1000000 == 0 ? 1 : 2;\n"
+"Last-Translator: Denis Bitouzé <dbitouze@wanadoo.fr>, 2024\n"
+"Language-Team: French (https://app.transifex.com/12rambau/teams/166811/fr/)\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=utf-8\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.13.0\n"
+"Language: fr\n"
+"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % 1000000 == 0 ? 1 : 2;\n"
```

Comment:
I wish I knew why transifex keeps introducing trivial changes like `utf` -> `UTF` or changing the order of these header/metadata lines. It makes these PRs much harder to assess.

Reply:
We can try to introduce a precomit-ci hook that normalise all those...

Reply:
```python
import polib
po = polib.pofile("sphinx.po")
po.metadata = {k: v for k, v in sorted(po.metadata.items())}
po.save("sphinx.po")
```

Should at least sort headers...

Reply:
Or see #1813